taskcluster/firefox_tv_taskgraph/__init__.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import absolute_import, print_function, unicode_literals
```

Comment:
nit: not sure what was decided by licenses for the files in this commit

Reply:
:/ right, I'll paste those in